docs/serving/app-security/security-guard-quickstart.md

```diff
@@ -0,0 +1,130 @@
+# Security-Guard Monitoring Quickstart
+
+Before starting the tutorial, learn [about Security-Guard](./security-guard-about.md)
+
+In this tutorial, deploy a Hello World service and protect it with Guard.
+
+## Before you begin
+
+Using Guard requires that your cluster will use an enhanced queue-proxy image.
+In addition, Guard include automation for auto-learning a per service Guardian. Auto-learning requires you to deploy a guard-service on your kubernetes cluster.
+
+To start this tutorial, after installing Knative Serving, run the following procedure to replace your queue-proxy image and deploy a guard-service in the current namespace.
+
+=== "Install from source"
+
+    1. Clone the security-guard repository using `git clone git@github.com:knative-sandbox/security-guard.git`
+
+    1. Do `cd security-guard`
+
+    1. Run `ko apply -Rf ./config`
+
+=== "Install from released images"
+
+    Use released images to update your system to enable Guard:
+
+    1. Replace the queue-sidecar-image with `ghcr.io/knative.dev/security-guard/guard-service`
+       for example if you are not using an operator, you may update config-deployment directly using 
+        `kubectl edit ConfigMaps -n knative-serving config-deployment` to use the released
+        queue-sidecar-image version that include Guard.
+
+        ```
+        apiVersion: v1
+        kind: ConfigMap
+        ...
+        data:
+            queue-sidecar-image: ghcr.io/knative.dev/security-guard/guard-service
+        ```
+
+        If you are not using an operator, you may use the following command to update teh config-deployment ConfigMap:
+        ```
+        kubectl apply -f https://raw.githubusercontent.com/knative-sandbox/security-guard/main/deploy/queue-proxy.yaml`
```

Comment:
I agree. Also, we don't have now /deploy/queue-proxy.yaml. 
Notice that this is outside of /config since in /config we use ko and here we are using the images produced during the release.